### PR TITLE
irresistible

### PR DESCRIPTION
**Pull request by Skrem7**
# Document the changes in your pull request

also makes vampire intro ten times more mentor-like

# Changelog

:cl:  
spellcheck: fixes a minor typo in lilith pact flavor text
spellcheck: Vampire intro now fully informs you how to succ
/:cl:

### CODE REVIEW

**Comment by Maia-J** [comment]
Lizard lisp moment

File: yogstation/code/datums/antagonists/vampire.dm [hunk 1]
```diff
@@ -263,7 +263,7 @@
 		O.visible_message(span_danger("[O] grabs [H]'s neck harshly and sinks in their fangs!"), span_danger("You sink your fangs into [H] and begin to [blood_to_take > BLOOD_SUCK_BASE ? "quickly" : ""] drain their blood."), span_notice("You hear a soft puncture and a wet sucking noise."))
 		playsound(O.loc, 'sound/weapons/bite.ogg', 50, 1)
 	else
-		to_chat(O, span_notice("You stealthily begin to drain blood from [H], be careful, as they will notice if their blood gets too low."))
+		to_chat(O, span_notice("You stealthily begin to drain blood from [H]. Be careful, as they will notice if their blood gets too low."))
```
Comment (ReddicusDragon) [inline yogstation/code/datums/antagonists/vampire.dm:266]:
```suggestion
		to_chat(O, span_notice("You stealthily begin to drain blood from [H]; be careful, as they will notice if their blood gets too low."))
```

File: yogstation/code/game/gamemodes/vampire/vampire_powers.dm [hunk 2]
```diff
@@ -454,7 +454,7 @@
 					to_chat(target, span_danger("Wicked shadows invade your sight, beckoning to you."))
 					to_chat(user, span_notice("We begin to drain [target]'s blood in, so Lilith can bless it."))
 				if(2)
-					to_chat(target, span_danger("Demonic whispers fill your mind, and they become irressistible..."))
```
Comment (ReddicusDragon) [inline yogstation/code/game/gamemodes/vampire/vampire_powers.dm:457]: a lizard wrote this

File: yogstation/code/datums/antagonists/vampire.dm (hunk 1 above)
Comment (ReddicusDragon) [reply yogstation/code/datums/antagonists/vampire.dm:266]: this can go either way

**Review by ReddicusDragon — APPROVED** [review]

File: yogstation/code/datums/antagonists/vampire.dm (hunk 1 above)
Comment (Skrem7) [reply yogstation/code/datums/antagonists/vampire.dm:266]: yes but i prefer a second sentence

**Pull request merged**